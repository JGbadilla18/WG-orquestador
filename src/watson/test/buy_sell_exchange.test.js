const testWatson = require('./testWatson');

watsonText = 'Aceptan dolares'

it(watsonText, () => {
  expect.assertions(1);
  return testWatson(watsonText).then(data => expect(data).toMatchSnapshot());
});
