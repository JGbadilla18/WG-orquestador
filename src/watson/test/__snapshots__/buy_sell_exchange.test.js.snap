// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Aceptan dolares 1`] = `
Object {
  "actions": Array [],
  "components": Array [
    Object {
      "buttonType": "web_url",
      "buttons": Array [
        Object {
          "title": "Aquí",
          "url": "https://blog.corporacionbi.com/solucionesbi/comprar-divisas-efectivo-agencias-multimoneda-banco-industrial",
        },
      ],
      "response_index": 0,
      "response_type": "button",
      "text": "Puedes verificar las monedas que ponemos a tu disposición y en qué Agencias Bi Multimoneda puedes encontrarlas",
    },
    Object {
      "response_type": "pause",
      "time": 10000,
      "typing": true,
    },
  ],
}
`;
